### ISSUE TITLE
Scanning multiple files v.likely to hang during scanning if Worker Process enabled (there is a workaround)

### ISSUE THREAD

**Issue by YupoWert** [RandomEngy/VidCoder#988]
### Problem Description

I have 150,000+ video clips I wish to process into x265.
I have an initial batch of 6000 clips

I have attempted multiple times to drag these in for scanning / queuing in ever decreasing smaller selections. Sometimes even dragging in just 50 files for scanning, will result in the scanning 'progress bar' hanging at some time. It stalls, becomes completely unresponsive, and cannot be closed. You have to END TASK Vidcoder to recover, and restart. If I re-drag to same 50 files it may process them, or hang at a different progression through completion.

HOWEVER - I have discovered a workaround.
If I DESELECT "Use worker process to encode" and restart Vidcoder, then I can successfully import all 6000 files in one dragging operation. I have now repeated both these scenarios with identical results on 2 separate PC's. Both times, deselecting "Use worker process to encode" has enabled a successful mass scanning without issue*. I then export the Queue, change Vidcoder back to ENABLE  "Use worker process to encode", reload, and start the encoding process.

(* I have raised a separate issue with regard to the popup "Error Scanning Video Files" window which cannot be resized / scrolled to cope with a list of failed scans taller than the display res)

[Logs.zip](https://github.com/RandomEngy/VidCoder/files/8131750/Logs.zip)

### What version of VidCoder are you running?

7.12

### Encode Log

```text
I have attached a ZIP file containing the entire logs folder.
Prior, I uninstalled all editions of Vidcoder, cleared out the LOGS folder
re-installed 7.12, and dragged in 100+ files.
After it became apparent of the progress bar had stalled, and had become unresponsive, I END TASK Vidcoder, and zipped up these logs.
```

**Comment by YupoWert** [comment]
Follow up:
I viewed the log files, which seem to indicate that the clip "Outdoors 39.mp4" is the last reported partially scanned file.
I can play the file within multiple media players without error.
I can drag this individual file into Vidcoder without error
Vidcoder will encode it into x265 without error.

As it is only a ~ 1 sec clip I have attached it. Many of the video clips in my list are sub 5 second. (The very small duration may seem odd, but it is often the only known clips of various passed family members in existence)

I have attached the individual scan log for that file, and a zip of the other logs for that.


[2022-02-24 10.59.56 Scan outdoors 039.txt](https://github.com/RandomEngy/VidCoder/files/8131944/2022-02-24.10.59.56.Scan.outdoors.039.txt)
[logs2.zip](https://github.com/RandomEngy/VidCoder/files/8131957/logs2.zip)

https://user-images.githubusercontent.com/73061808/155512904-8334df8e-6769-4c5a-ad77-0f4433af0f73.mp4
